packages/erlang/erlang.0.0.13/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Libraries to manipulate Erlang sources"
+description: """
+erlang is a  a set of libraries designed to facilitate manipulating
+ Standard Erlang and Core Erlang sources.
+
+ It provides a lexer/parser, a concrete AST, and a printer for Standard Erlang
+ in its current version.
+"""
+maintainer: ["Leandro Ostera <leandro@ostera.io>"]
+authors: ["Leandro Ostera <leandro@ostera.io>"]
+license: "Apache-2.0"
+homepage: "https://github.com/AbstractMachinesLab/caramel"
+bug-reports: "https://github.com/AbstractMachinesLab/caramel/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2.7"}
```

Comment:
Looks good overall, I think this is dune trying to be smart but not enough with constraints 
```suggestion
  "dune" {>= "2.7"}
```